draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -1150,6 +1150,8 @@ and should monitor suspicious behaviour such as frequent rotation of those ident
 If such behaviour is detected, Verifiers are advised to reject SD-JWT VCs issued by those
 Issuers.
 
+Another related concern arises from the use of confirmation methods in the cnf claim that involve retrieving key material from a remote source, especially if that source is controlled by the issuer. This includes, but is not limited to, the use of the x5u parameter in JWKs ([RFC7517, section 4.6]), the jku parameter ([RFC7800, section 3.5]), and cases where a URL is used in the kid parameter ([RFC7800, section 3.4]). Future confirmation methods may also introduce remote retrieval mechanisms. Issuers are advised not to issue SD-JWT VCs with such cnf methods, and verifiers and holders are advised not to follow or resolve remote references for key material in the cnf claim. Only confirmation methods that do not require remote retrieval of key material should be supported.
```

Comment:
I suggest using `SHOULD`, not `should` in "key material should be supported."

Just like https://datatracker.ietf.org/doc/html/rfc8725#section-3.10 does it.

Reply:
In addition to that, rephrase or omit "Issuers are advised not to issue SD-JWT VCs with such cnf methods".

Reply:
@cre8, my (personal) preference would be to explicitly block `x5u`, `jku`, and other feature methods for retrieving key material from a remote source in the cnf claim. The right place for that probably isn't going to be the privacy considerations section like you said.

Why is that?

This is a safeguard from anyone trying to use these methods for rotating key material, and of course, that eliminates a ton of privacy concerns.

Otherwise, there’s enough room left for misuse, abuse and debates.

Reply:
Sounds valid to me, but as mentioned above it would be better in the sd-jwt kb-jwt chapter that we can not change. So we either forbid it in the privacy concern chapter or adding a new kb-jwt chapter that is "hardening" the approach.
@bc-pi what's your opinion?